setup.py

```diff
@@ -18,4 +26,4 @@ def generate_requirements():
       author_email='ybronhei@redhat.com',
       url="https://github.com/emesika/RaananaTiraProject",
       scripts=["rose-client", "rose-server", "rose-admin"],
-      data_files=[('requirements.txt', [generate_requirements()]), ])
+      data_files=DATA_FILES)
```

Comment:
Can we specify data_files always, and generate the file only when running sdist?